alert-platform/build.gradle

```diff
@@ -16,10 +16,10 @@ dependencies {
         api 'com.sun.mail:javax.mail:1.6.2'
         api 'javax.mail:javax.mail-api:1.6.2'
 
-        api 'com.synopsys.integration:integration-rest:10.3.6'
+        api 'com.synopsys.integration:integration-rest:10.3.9-SNAPSHOT'
         api 'com.synopsys.integration:integration-common:26.0.4'
         api 'com.synopsys.integration:blackduck-common:66.0.2'
-        api 'com.synopsys.integration:int-jira-common:3.0.7'
+        api 'com.synopsys.integration:int-jira-common:3.0.8-SNAPSHOT'
```

Comment:
We shouldn't merge in the SNAPSHOT versions, I think these should be updated with the released versions of the libraries once the releases go out.